src/uct/api/v2/uct_v2.h

```diff
@@ -958,6 +960,7 @@ ucs_status_t uct_md_query_v2(uct_md_h md, uct_md_attr_v2_t *md_attr);
  * @return                  Error code.
  */
 ucs_status_t uct_md_mkey_pack_v2(uct_md_h md, uct_mem_h memh,
+                                 void *address, size_t length,
```

Comment:
1. Not sure what memory to expose means here.
2. are these optional parameters ?

Reply:
1. It means exposing a remote access by the packed remote key.
2. No, address and length are fast-path and mandatory, we want to pass them by registers (when possible), they are going to be used in RNDV protocol flow

Reply:
Can you please clarify why do need to expose particular address  and length ? Why this is needed now and it worked fine before without those args.

Reply:
We've expanded ucp_pack_rkey to include the address and length. This adjustment is necessary because the region we register with UCP for CUDA IPC might encompass the CUDA memory address space, which wasn't initially allocated. This situation arises, especially when aligning region boundaries to the GPU page size in GDR copy usage.
To avoid failures caused by calling cuIpcGetMemHandle() for non-allocated memory, we defer this call until a later stage when ucp_pack_rkey is invoked for a particular memory operation. At this point, we have the precise address and length required for CUDA IPC access, unaffected by rcache alignment.

Reply:
got it . thanks